app/policies/wiki_policy.rb

```diff
@@ -14,14 +14,17 @@ def index?
 
   def edit?
     user.present?
+     @wiki.private != true || @wiki.user == @user
```

Comment:
`@wiki.private` itself returns true or false so you could do `!@wiki.private` to negate the value without comparing.